source/learners/sfd_discussions/getting_started.rst

```diff
@@ -0,0 +1,91 @@
+.. _Getting started with Discussions:
+
+##################################
+Getting started with Discussions
+##################################
+
+This section describes where to find discussions and the structure and elements of course discussions.
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+.. _Where to Find Discussions:
+
+**************************
+Where to find Discussions
+**************************
+
+You can access course discussions in two ways:
+
+1. From the course navigation
+
+   Click the **Discussions** tab at the top of your course. This opens the full discussion page, 
+   where you can browse all topics, search, and post new questions or discussions.
+
+   .. figure:: /_images/learners/Discussions_course_nav.png
+      :alt: The Discussions page can be accessed from the course navigation bar.
+      :width: 600px
+      :align: center
+
+      The Discussions page can be accessed from the course navigation bar.
+   
+
+2. From within a course unit
+
+   Some course units include a discussion panel on the right side of the page.  
+   You can use this panel to read or take part in discussions related specifically to that unit.
+
+   If you don’t see this panel, it may be because:
+
+   a. Discussions aren’t enabled for that unit. This is common for assignments or exams.  
+      You can still use the main *Discussions* tab to participate in other discussions.
+
+   b. The panel is collapsed. Look for a button in the upper-right corner of the page  
+      (as shown in the image below). Click it to expand the discussion panel.
+
+   Remember that this panel only includes posts related to that particular unit.  
+   You can learn how to browse and search across all course discussions on the next page.
+
+   .. figure:: /_images/learners/Discussions_sidebar.png
+      :alt: Screenshot of discussions panel on the right.
+      :width: 600px
+      :align: center
+
+      The Discussions panel appears on the right side of course content when enabled.
+
+
+.. _Basic elements:
+
+***************
+Basic elements
+***************
+
+Each discussion has the following components:
+
+#. Post: The question or idea that someone shares. A post starts a discussion. It has a title, name of the author, its type (question or discussion) and the topic it is linked to.
```

Comment:
[possible idea] bold the type, to make scannability a bit easier

```suggestion
#. **Post**: The question or idea that someone shares. A post starts a discussion. It has a title, name of the author, its type (question or discussion) and the topic it is linked to.
```

Reply:
Done